test_binheap.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+
+def test_pop():
+    """Test if entry sorts to the correct position."""
+    from binheap import Heap
+    heap = Heap()
+    heap.container = [2, 3, 4, 10]
+    heap.pop()
+    assert heap.container == [10, 3, 4]
+
+
+def test_push():
+    """Test when a value is appended to the heap the structure is maintained."""
+    from binheap import Heap
+    heap = Heap()
+    heap.container = [4, 7, 38, 96]
+    heap.push(5)
+    assert heap.container == [96, 7, 38, 4, 5]
+
```

Comment:
Make sure to add more tests that cover edge cases. We want to see that your data structure functions correctly at edge cases like popping from an empty heap or a heap with 1 item.